modules/data-entry/src/main/frontend/src/vocabQuery/VocabularyBranch.jsx

```diff
@@ -193,7 +193,7 @@ function ListChild(props) {
 
       {/* Term name */}
       <Typography onClick={() => loadTerm(id, path)}
-                  className={classes.infoName + (bolded ? (" " + classes.boldedName) : " ")}>
+                  className={classes.infoName + (focused ? (" " + classes.boldedName) : " ")}>
```

Comment:
Rename the class name:
```suggestion
                  className={classes.infoName + (focused ? (" " + classes.focusedTermName) : " ")}>
```